src/languageSupport/languages/flux/lsp/utils.ts

```diff
@@ -0,0 +1,61 @@
+import {
+  ProtocolRequestType,
+  ProtocolNotificationType,
+} from 'vscode-languageserver-protocol'
+
+export const makeRequestType = (method: string) => {
+  return new ProtocolRequestType<any, any, any, any, any>(method)
+}
+
+export const makeNotificationType = (method: string) => {
+  return new ProtocolNotificationType<any, any>(method)
+}
+
+const JSONRPC = '2.0',
+  FLUXLANGID = 'flux'
+
+const createNotification = (method: string, params: object = {}) => {
+  return {
+    jsonrpc: JSONRPC,
+    method,
+    params,
+  }
+}
+
+export const didChange = (uri: string, text: string, version: number) => {
+  return createNotification(Methods.didChange, {
+    contentChanges: [{text}],
+    textDocument: {
+      uri,
+      version,
+    },
+  })
+}
+
+export const didOpen = (uri: string, text: string, version: number) => {
+  return createNotification(Methods.didOpen, {
+    textDocument: {
+      uri,
+      languageId: FLUXLANGID,
+      version,
+      text,
+    },
+  })
+}
+
+export enum Methods {
```

Comment:
I can probably import these methods instead, from one of the support libs.
Will update tmrw.

Reply:
Looking at the src for `vscode-languageserver-node`, looks like they also just hardcode the strings. Therefore, going to follow the same pattern and not import.